test.py

```diff
@@ -91,4 +91,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main(argv[1:])
```

Comment:
This is personal preference, but let's not make `main()` take in arguments.

Reply:
5599c8e